### PR TITLE
Limit account statement export

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Constrain the account statement export to avoid memory blowing up.

## Changes

- backend: the export statement REST call now limit the chosen period to 32 days, and defaults fromTime to 31 days ago and toTime to now.
- frontend: added a button to choose a month to export statement for.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by shjortConcordium** [comment]
I'm not sure who the "ccdscan team" is, so sorry if I missed someone or added anyone who is not.

I just need someone to review it :upside_down_face:

File: frontend/src/components/Accounts/AccountDetailsAccountStatement.vue [hunk 1]
```diff
@@ -133,13 +133,20 @@
...
 					<span class="hidden md:inline">Export</span>
 					<DownloadIcon class="h-4 inline align-text-top" />
 				</a>
 			</div>
 		</div>
+		<div class="col-span-1 flex justify-end">
+			<input
+				v-model="chosenMonth"
+				class="bg-theme-background-primary-elevated-nontrans rounded"
+				type="month"
```
Comment (soerenbf) [inline frontend/src/components/Accounts/AccountDetailsAccountStatement.vue:147]: Would be good to set `min` (date of account creation or chain genesis if not feasible) and `max` (current date) properties to limit the range of months you can select.

**Review by limemloh — APPROVED** [review]

File: frontend/src/components/Accounts/AccountDetailsAccountStatement.vue (hunk 1 above)
Comment (shjortConcordium) [reply frontend/src/components/Accounts/AccountDetailsAccountStatement.vue:147]: Added.

File: frontend/src/components/Accounts/AccountDetailsAccountStatement.vue [hunk 2]
```diff
@@ -133,13 +133,22 @@
...
 					class="bg-theme-button-primary px-8 py-3 hover:bg-theme-button-primary-hover rounded"
-					:href="exportUrl"
+					:href="exportUrl(accountAddress, chosenMonth)"
 				>
 					<span class="hidden md:inline">Export</span>
 					<DownloadIcon class="h-4 inline align-text-top" />
 				</a>
 			</div>
 		</div>
+		<div class="col-span-1 flex justify-end">
```
Comment (soerenbf) [inline frontend/src/components/Accounts/AccountDetailsAccountStatement.vue:143]:
```suggestion
		<div class="col-span-1 flex justify-end mt-3">
```

**Review by soerenbf — APPROVED** [review]

File: frontend/src/components/Accounts/AccountDetailsAccountStatement.vue [hunk 3]
```diff
@@ -133,13 +133,22 @@
...
+			<input
+				v-model="chosenMonth"
+				class="bg-theme-background-primary-elevated-nontrans rounded"
```
Comment (soerenbf) [inline frontend/src/components/Accounts/AccountDetailsAccountStatement.vue:146]:
```suggestion
		<div class="col-span-1 flex justify-end mt-3">
			<input
				v-model="chosenMonth"
				class="bg-theme-background-primary-elevated-nontrans rounded py-2 px-4"
```

**Pull request merged**